jenkinsfiles/ginkgo-gke.Jenkinsfile

```diff
@@ -63,11 +63,6 @@ pipeline {
                 sh 'ls -A | grep -v src | xargs mv -t ${PROJ_PATH}'
             }
         }
-        stage('Start docker registry for build') {
-            steps{
-                sh 'cd ${TESTDIR}/gke; ./start-registry.sh'
```

Comment:
It looks like the `start-registry.sh` script is no longer used after this PR. Should it be removed as well?